Samples/PhotoEditor/cpp-winui-packaged/PhotoEditor/DetailPage.idl

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Consider consolidating all `.idl` files in a `Project.idl` like in [the template file](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/Templates/VSIX/Project%20Templates/cpp-winui-template/Project.idl)

Reply:
Fixed